compositor_render/src/renderer.rs

```diff
@@ -187,7 +186,19 @@ impl WgpuCtx {
             None,
         ))?;
 
-        Ok(Self { device, queue })
+        let yuv_bind_group_layout = YUVTextures::new_bind_group_layout(&device);
+        let rgba_bind_group_layout = RGBATexture::new_bind_group_layout(&device);
+        let yuv_to_rgba_converter = YUVToRGBAConverter::new(&device, &yuv_bind_group_layout);
+        let rgba_to_yuv_converter = RGBAToYUVConverter::new(&device, &rgba_bind_group_layout);
+
+        Ok(Self {
+            device,
```

Comment:
I'm not a fan of dumping everything in context like that, but for now I don't have any better idea